src/main/java/com/zemnov/salon/controller/UserController.java

```diff
@@ -0,0 +1,94 @@
+package com.zemnov.salon.controller;
+
+import com.zemnov.salon.model.Contact;
+import com.zemnov.salon.model.Role;
+import com.zemnov.salon.model.User;
+import com.zemnov.salon.repository.ContactRepo;
+import com.zemnov.salon.repository.UserRepo;
+import com.zemnov.salon.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Controller
+@RequestMapping("/user")
+@PreAuthorize("hasAuthority('ADMIN')")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @GetMapping
+    public String userList(Model model) {
+        List<User> users = userService.findAllUsers();
+        model.addAttribute("users", users);
+        return "userList";
+    }
+
+    @GetMapping("{user}/edit")
+    public String userEditForm(@PathVariable User user, Model model) {
+        model.addAttribute("user", user);
+        model.addAttribute("roles", Role.values());
+
+        return "userEdit";
+    }
+
+    @GetMapping("{user}/delete")
+    public String userDeleteForm(@PathVariable User user, Model model) {
+        model.addAttribute("user", user);
+        model.addAttribute("roles", Role.values());
+        return "userDelete";
+    }
+
+
+    @PostMapping("/delete")
+    public String userDelete(
+            @RequestParam("userId") User user
```

Comment:
Let's respect one style of code style, inline method parameters when it is possible